src/se/vidstige/jadb/DeviceWatcher.java

```diff
@@ -0,0 +1,44 @@
+package se.vidstige.jadb;
+
+import java.util.List;
+import java.io.IOException;
+
+public class DeviceWatcher implements Runnable {
+    private Transport transport;
```

Comment:
I use `transport` for checking if `stop` is called instead of `stopped`, so `transport` is mutable.
I should use both `stopped` and `transport`?